biofefi/components/plots.py

```diff
@@ -13,11 +13,12 @@ def plot_box(plot_dir: Path, box_title: str):
         plot_dir (Path): The directory containing the plots.
         box_title (str): The title of the plot box.
     """
-    plots = list(plot_dir.iterdir())
-    with st.expander(box_title, expanded=len(plots) > 0):
-        for p in plots:
-            if p.name.endswith(".png"):
-                st.image(str(p))
+    if plot_dir.exists():
```

Comment:
If you're doing the test here, does that mean you now don't need to test it in 4_Train_Models.py?